README.md

```diff
@@ -273,3 +274,29 @@ puts Create["pipetree"].inspect(style: :rows) #=>
  # 2 ===============================>model!
 ```
 This is helpful if you maintain modules for operations with generic steps.
+
+## Each Macro
+This macro let you wrap steps in each itereator. It's usefull when you want iterate through an array.
+
+```ruby
+class Multiplication < Trailblazer::Activity::Railway
+  step :init_result_array
+
+  # source -> ctx key where array is stored
+  # target -> ctx key where each element will be placed
+  step Each(source: :array, target: :element) {
```

Comment:
@btolarz Instead of fixing `source` to only top level, we should accept it from any deeply nested level too (for example, `params[:souce]`).

To handle such cases, how about we accept a callable entity (method, proc etc) and get the iterable at runtime ?

```ruby
Each(:users) # def users(ctx, **) ...
Each(->(ctx, params:, **) { User.where(id: params[:id]) })
```